ores/applications/util.py

```diff
@@ -0,0 +1,27 @@
+import glob
+import os.path
+import yamlconf
+import logging.config
+
+DEFAULT_DIRS = ["config/", "/etc/ores/"]
```

Comment:
Because, generally, these applications will be configured via uwsgi or celery worker and will not be called as command-line utilities.  See my notes on the initial PR.